repo/fsrepo/fsrepo_test/README.md

```diff
@@ -0,0 +1 @@
+This directory is used to store FSRepos generated during go tests.
```

Comment:
why not Mktemp -- http://golang.org/pkg/io/ioutil/#TempDir? (dont care much either way). Oh i guess we can inspect them if they're here. Worried this would make them not able to run concurrently though, which is painful.